googlemaps/places.py

```diff
@@ -396,7 +396,14 @@ def _places(
     return client._request(url, params)
 
 
-def place(client, place_id, session_token=None, fields=None, language=None):
+def place(
+    client,
+    place_id,
+    session_token=None,
+    fields=None,
+    language=None,
+    reviews_sort=None
```

Comment:
In keeping with [the API](https://developers.google.com/maps/documentation/places/web-service/details#reviews_sort), this can default to "most_relevant"